actors/builtin/miner/miner_state.go

```diff
@@ -917,8 +926,8 @@ func (st *State) AssertBalanceInvariants(balance abi.TokenAmount) {
 }
 
 func (st *State) MeetsInitialPledgeCondition(balance abi.TokenAmount) bool {
-	available := st.GetUnlockedBalance(balance)
-	return available.GreaterThanEqual(st.InitialPledgeRequirement)
+	unlockedBalance := st.GetUnlockedBalance(balance)
+	return unlockedBalance.GreaterThanEqual(st.InitialPledgeRequirement)
```

Comment:
> Note: This is policy, rather than pure state, and so perhaps belongs in actor code or a free function rather than a state method. 

I can get on board with that

> I don't think we want to say a miner is meeting their pledge condition if fee debt is nonzero.

I mildly disagree.  As `verifyPledgeRequirementsXXX` expands in scope to include constraints on recent consensus faults and potentially consensus fee related balance I think it helps clarity to unravel eligibility requirements into a set with no blurry overlap.  Having a condition for "unlocked balance fulfills pledge requirement" and another for "fee debt is zero" and then combining the two where needed feels better.

Reply:
Resolution, IP reqs only met if fee debt is zero.